### DISCUSSION TITLE
Is possible apply different authencaiton flows to different clients in a realm?

### DISCUSSION THREAD

**Discussion by hantsy** [keycloak/keycloak#30726] [Q&A]
For example, I would like to use a username/password(OIDC authorization code flow) for the administration dashboard and a further email OPT validation step.  

But for the end user's web application, I want to use a simple SMS OPT flow for the authentication.

**Comment by BoThomas-E** [comment]
I was looking into this as well.
Found the `Authentication flow overrides` in the clients `Advanced` settings at the bottom (Version `25.0.2`).
Hope this helps 🙌

**Reply by col-panic** [reply]
yes, but it seems, that its not possible to override every flow, e.g. `Client authentication flow` cannot be overriden.

**Reply by Sergej417** [reply]
Is this still the case, that Client authentication flow cannot be overridden?

**Reply by col-panic** [reply]
@Sergej417 not sure, I think I finally found a solution. There has to be an issue or discussion or seomething else, I remember discussing the topic and the solution being not that obvious.

**Reply by Sergej417** [reply]
@col-panic Thank you! Any idea how should i search for it, i am struggling to find it in the issues here 😅

**Reply by col-panic** [reply]
@Sergej417 maybe this https://github.com/keycloak/keycloak/issues/41562
